src/common/components/nav/navCart/navCart.component.spec.js

```diff
@@ -133,4 +133,16 @@ describe('navCart', () => {
       expect($ctrl.$window.location).toBe('https://give-stage2.cru.org/checkout.html')
     })
   })
+
+  describe('buildCartUrl', () => {
+    it('should build a url without query parameters', () => {
+      expect($ctrl.buildCartUrl()).toEqual('https://give-stage2.cru.org/cart.html')
+    })
+
+    it('should build a url with query parameters', () => {
+      delete window.location
```

Comment:
Is this `delete` line needed here and in `src/common/services/api/cart.service.spec.js`?

Reply:
Yeah, this is what allows us to set `window.location` in the next line.

Reply:
OK. I would have thought you could just overwrite `window.location` like you're doing on the next line.